040-SDK/104-delete.mdx

```diff
@@ -127,7 +127,24 @@ users, _ := client.Transaction(context.TODO(), xata.TransactionRequest{
 
 </TabbedCode>
 
-[Transactions](/docs/sdk/transaction) will not fail a delete operation if no record is found. All operations in a transaction must succeed otherwise it is rolled back.
+To delete multiple records we recommend using transactions. [Transactions](/docs/sdk/transaction#delete) will not fail a delete operation if a record is not found. All operations in a transaction must succeed otherwise it is rolled back.
```

Comment:
```suggestion
To delete multiple records from the API we recommend using transactions. [Transactions](/docs/sdk/transaction#delete) will not fail a delete operation if a record is not found. All operations in a transaction must succeed otherwise it is rolled back.
```